lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -32,7 +32,7 @@ class AxeAudit extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    const violations = artifacts.Accessibility.violations || [];
+    const violations = artifacts.Accessibility.violations;
```

Comment:
now asserted as an array in the gatherer since [axe says](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#results-object) any valid result will include the `violations` array.